slatedb/src/compactor.rs

```diff
@@ -373,12 +382,9 @@ impl CompactorEventHandler {
         self.log_compaction_state();
         self.write_manifest_safely()?;
         self.maybe_schedule_compactions()?;
-        self.stats.last_compaction_ts.set(
-            SystemTime::now()
-                .duration_since(UNIX_EPOCH)
-                .unwrap_or_default()
-                .as_secs(),
-        );
+        self.stats
+            .last_compaction_ts
+            .set(self.clock.elapsed().as_secs());
```

Comment:
this is throwing me off a bit - why is there a separate `elapsed` method that basically returns the same thing as the `now`. This clock doesn't have a "starting point" like a traditional stopwatch/timer. It seems like we should just use clock::now_systime and create the duration here if we're just using it as a fancy way to convert to seconds